packages/loot-core/src/server/sync/proto/sync_pb.js

```diff
@@ -13,7 +13,7 @@
 
 var jspb = require('google-protobuf');
 var goog = jspb;
-var global = globalThis;
+var global = (function() { return this || window || global || self || Function('return this')(); }).call(null);
```

Comment:
I believe this was explicitly changed at one point to remove the usage of eval (`[new] Function('return this')()`)